system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -329,11 +328,8 @@ parameter_types! {
 
 impl pallet_transaction_payment::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type OnChargeTransaction = impls::tx_payment::FungiblesAdapter<
```

Comment:
Reasoning for this change: Using `FungiblesAdapter` (plural) for `NativeAndAsset` and `DotLocation` is the same as using `FungibleAdapter` (singular) with `Balances` because `NativeAndAsset` will just forward all calls to `Balances` for `DotLocation`.